migrations/versions/a28f2733897b_.py

```diff
@@ -0,0 +1,57 @@
+"""Sequence for cusstomuserattribute, tokengroup and tokentokengroup
```

Comment:
Please add a version prefix here. I think it is "v3.8: ...."

Reply:
79cdc78